src/components/RegisterForm.js

```diff
@@ -0,0 +1,138 @@
+                data-testid="email" required/>
+            
+            <div className="input-holder">
+                <input id="street" type="text" placeholder="Ulica" 
+                    className={`${isValid.street?"":"invalid-input"}`}
+                    value={user.street} 
+                    onChange={event => setUser({...user, street: event.target.value})}
+                    onMouseDown={(event) => setIsValid({...isValid, street: true})}
+                    data-testid="street" required/>
+
+                <input id="number" type="text" placeholder="Numer"
+                    className={`${isValid.number?"":"invalid-input"}`} 
+                    value={user.number} 
+                    onChange={event => setUser({...user, number: event.target.value})}
+                    onMouseDown={(event) => setIsValid({...isValid, number: true})}
+                    data-testid="number" required/>
+                
+                <input id="city" type="text" placeholder="Miasto"
+                    className={`${isValid.city?"":"invalid-input"}`}
+                    value={user.city} 
+                    onChange={event => setUser({...user, city: event.target.value})}
+                    onMouseDown={(event) => setIsValid({...isValid, city: true})}
+                    data-testid="city" required/>
+
+                <input id="post_code" type="text" placeholder="Kod pocztowy"
+                    className={`${isValid.post_code?"":"invalid-input"}`}
+                    value={user.post_code} 
+                    onChange={event => setUser({...user, post_code: event.target.value})} 
+                    onMouseDown={(event) => setIsValid({...isValid, post_code: true})}
+                    data-testid="post_code" required/>
+
+            </div>
+            
+            <input id="pass" type="password" placeholder="Hasło"
+                className={`${isValid.pass?"":"invalid-input"}`}
+                value={user.pass} 
+                onChange={event => {
+                        setUser({...user, pass: event.target.value})
+                        validatePass(event.target.value) ? setPassStr(true):setPassStr(false)
+                        passConf===event.target.value  && setPassDiff(false)
+                }} data-testid="pass" required/>           
+            {!passStr && <p className="invalid">Hasło jest za słabe!</p>}
+            
+            
+            <input id="pass-confirmation" type="password" placeholder="Powtórz hasło" 
+                value = {passConf} onChange={event => {
+                    setPassConf(event.target.value)
+                    event.target.value!==user.pass ? setPassDiff(true) : setPassDiff(false)}
+                 } data-testid="pass-confirmation" required/>
+            
+            {passDiff && <p className="invalid">Hasła się nie zgadzają!</p>}
+            {invalid && <p className="invalid">Jedno lub więcej pól jest nieprawidłowych!</p>}
```

Comment:
To jest za mało jezeli chodzi o user expirience. Kazde pole powinno miec info walidacyjne, co konkretnie jest nie tak.